src/tests/global_variables/default_variables.py

```diff
@@ -1,39 +1,57 @@
 from contextlib import suppress
-from typing import Any
+from typing import Any, ClassVar
 
 from assisted_test_infra.test_infra.helper_classes.config.base_config import Triggerable
-from assisted_test_infra.test_infra.utils import utils
+from assisted_test_infra.test_infra.utils import EnvVar, utils
 from service_client import ClientFactory, InventoryClient
-from tests.global_variables.env_variables_defaults import _EnvVariablesDefaults
-from tests.global_variables.triggers import get_default_triggers
+from tests.global_variables.env_variables_defaults import _EnvVariables
+from triggers import Trigger, get_default_triggers
 
 
-class DefaultVariables(_EnvVariablesDefaults, Triggerable):
+class DefaultVariables(_EnvVariables, Triggerable):
+    __instance: ClassVar = None
+
+    def __new__(cls, *args, **kwargs):
+        """Prevent creating another env_var instance"""
+        if isinstance(cls.__instance, cls):
+            raise Exception("Can't initialized more then one global configuration object")
+        cls.__instance = object.__new__(cls, *args, **kwargs)
+        return cls.__instance
+
+    def __getattribute__(self, item):
```

Comment:
also here, possible ``__getattr__`` will be simpler to use

Reply:
Here we can't do it. We want to return the value of the env and not the env itself (and only if it's of type .
When loading the configurations on configuration.__init__ we try to load the defaults. 
I'll document it